pkg/server/singleprocess/service_ui_pipeline.go

```diff
@@ -0,0 +1,81 @@
+package singleprocess
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+	"github.com/hashicorp/waypoint/pkg/server/hcerr"
+	serverptypes "github.com/hashicorp/waypoint/pkg/server/ptypes"
+)
+
+func (s *Service) UI_ListPipelines(
+	ctx context.Context,
+	req *pb.UI_ListPipelinesRequest,
+) (*pb.UI_ListPipelinesResponse, error) {
+	if err := serverptypes.ValidateUIListPipelinesRequest(req); err != nil {
+		return nil, err
+	}
+
+	// Create uninitialized array of pipeline bundles
+	var allPipelines []*pb.UI_PipelineBundle
+
+	// Get list of all pipelines
+	pipelineListResponse, err := s.state(ctx).PipelineList(ctx, req.Project)
+	if err != nil {
+		return nil, hcerr.Externalize(
+			hclog.FromContext(ctx),
```

Comment:
Tiny nit, no need to update this PR: 

I'm a fan of adding `log := hclog.FromContext(ctx)` to the top of these functions, then just using `log` going forward. It's more concise overall, and makes it easier to [add things](https://github.com/hashicorp/go-hclog#create-a-new-logger-with-fixed-keyvalue-pairs) to the logger throughout the function.